app/components/project_dashboard/information_component.html.erb

```diff
@@ -0,0 +1,43 @@
+<p class="py-2">
+  <% if @project.description && !@project.description.blank? %>
+    <%= @project.description %>
+  <% else %>
+    <%= t(:"components.project_dashboard.information.no_description") %>
+  <% end %>
+</p>
+
+<hr class="h-px my-4 bg-gray-200 border-0 dark:bg-gray-700"/>
+
+<span class="flex items-center">
+  <%= viral_tooltip(title: t(:"components.project_dashboard.information.number_of_samples")) do %>
+    <span class="flex items-center">
+      <%= viral_icon(name: :squares_2x2, color: :subdued, classes: "h-5 w-5 m-0 mr-2") %>
```

Comment:
**nitpick (blocking):** Can you change this to the flask icon that is used elsewhere to denote samples.

Reply:
Fixed in 24d7fa9977a9919ec1c5d1cb43205c9ca9af9db3